shared/login/relogin/index.desktop.tsx

```diff
@@ -51,89 +52,150 @@ class Login extends React.Component<Props, State> {
   render() {
     const inputProps = {
       autoFocus: true,
-      errorText: this.props.inputError ? this.props.error : '',
+      errorText: this.props.inputError,
       floatingHintTextOverride: '',
+      hideLabel: true,
       hintText: 'Password',
+      inputStyle: styles.passwordInput,
       key: this.props.inputKey,
       onChangeText: password => this.props.passwordChange(password),
       onEnterKeyDown: () => this.props.onSubmit(),
       ref: this._inputRef,
+      style: styles.passwordInputContainer,
       type: this.props.showTyping ? 'passwordVisible' : 'password',
       uncontrolled: true,
     } as const
 
-    const checkboxProps = [
-      {
-        checked: this.props.showTyping,
-        label: 'Show typing',
-        onCheck: check => {
-          this.props.showTypingChange(check)
-        },
-      } as const,
-    ]
-
     const userRows = this.props.users
       .concat({hasStoredSecret: false, username: other})
       .map(u => <UserRow user={u.username} key={u.username} />)
 
     const selectedIdx = this.props.users.findIndex(u => u.username === this.props.selectedUser)
 
     return (
-      <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true}>
-        {this.props.bannerError && (
-          <Kb.Banner color="red">
-            <Kb.BannerParagraph bannerColor="red" content={this.props.error} />
-          </Kb.Banner>
-        )}
-        <Kb.Box style={stylesContainer}>
-          <Kb.UserCard username={this.props.selectedUser}>
+      <SignupScreen
+        banners={errorBanner(this.props.bannerError)}
+        headerStyle={styles.header}
+        rightActionComponent={
+          <Kb.Button
+            type="Default"
+            mode="Secondary"
+            onClick={this.props.onSignup}
+            label="Create an account"
+          />
+        }
+        title="Log in"
+      >
+        <Kb.Box2 direction="vertical" fullHeight={true} fullWidth={true} style={styles.contentBox}>
+          <Kb.UserCard
+            username={this.props.selectedUser}
+            outerStyle={styles.container}
+            style={styles.userContainer}
+          >
             <Kb.Dropdown
               onChanged={this._onClickUser}
               selected={userRows[selectedIdx]}
               items={userRows}
               position="bottom center"
+              style={styles.userDropdown}
             />
-            <Kb.FormWithCheckbox
-              style={{alignSelf: 'stretch'}}
-              inputProps={inputProps}
-              checkboxesProps={checkboxProps}
-            />
-            <Kb.WaitingButton
-              disabled={!this.props.password}
+            <Kb.Box2 direction="horizontal" fullWidth={true} style={styles.inputRow}>
+              <Kb.Input {...inputProps} />
```

Comment:
new mocks use Kb.LabeledInput

Reply:
That excludes `passwordVisible`, is that ok?

Reply:
Also `LabeledInput` has buggy behavior. The user dropdown doesn't properly hover/click on a name that overlays above the `LabeledInput`, even when I change the `inputRef` to refer to a `LabeledInput`.